docs/reference/search/search-your-data/ingest-vectors.asciidoc

```diff
@@ -0,0 +1,131 @@
+[[bring-your-own-vectors]]
+=== Bring your own vector embeddings to {es}
+++++
+<titleabbrev>Bring your own vector embeddings</titleabbrev>
+++++
+
+This tutorial demonstrates how to index documents that already have dense vector embeddings into {es}.
```

Comment:
Is it worth adding an example for `sparse_vector` embeddings here as well?

Reply:
I think it would be best to keep this tightly focused to dense vectors and investigate demand for sparse vector equivalent going forward.